dist/common/scripts/scylla_tune_sched

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+#
+# Copyright 2023-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import os
+import sys
+
+PROCFS = '/proc/sys/kernel'
+DEBUGFS = '/sys/kernel/debug'
+
+TUNE_PARAMS = {
+    # Prevent auto-scaling from doing anything to our tunables
+    'sched.tunable_scaling': 0,
+
+    # Preempt sooner (For CFS, only available on <linux-6.6)
+    'sched.min_granularity_ns': 500000,
+
+    # Preempt sooner  (For EEVDF, only available on >=linux-6.6)
+    'sched.base_slice_ns': 500000,
+
+    # Don't delay unrelated workloads (For CFS, only available on <linux-6.6)
+    'sched.wakeup_granularity_ns': 450000,
+
+    # Schedule all tasks in this period (For CFS, only available on <linux-6.6)
+    'sched.latency_ns': 1000000,
+
+    # autogroup seems to prevent sched_latency_ns from being respected
+    'sched.autogroup_enabled': 0,
+
+    # Disable numa balancing
+    'numa_balancing': 0
+}
+
+def write_to(prefix, key, value):
+    if prefix == PROCFS:
+        path = os.path.join(prefix, key.replace('.', '_'))
+    elif prefix == DEBUGFS:
+        path = os.path.join(prefix, key.replace('.', '/'))
+    if not os.path.isfile(path):
+        return False
+    with open(path, 'w') as f:
+        f.write(str(value))
```

Comment:
This could fail (example: in a container, without elevated permissions). So need to catch the exception and return False.